### ISSUE TITLE
Pylint 0.28 reports a false positive E1103 under Python 3.3

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#9]
Originally reported by: **Chris Wong (BitBucket: [wongc](http://bitbucket.org/wongc), GitHub: @wongc?)**

---

Given files: t.py, tmp/u.py and tmp/v.py (and of course an empty tmp/____init____.py)

**t.py**

```
#!python

import tmp.u
tmp.u.obfuscate()
```

**tmp/u.py**

```
#!python

from .v import *
```

**tmp/v.py**

```
#!python

def obfuscate():
    print("hello world")
```

I will always get a E1101 error under Python 3.3 for t.py

```
#!

************* Module t
C0111:  1,0: Missing docstring
E1101:  4,0: Module 'tmp.u' has no 'obfuscate' member
```

This works fine using Pylint under 2.6.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/9
